crates/sui-framework/tests/delegation_tests.move

```diff
@@ -165,8 +165,7 @@ module sui::delegation_tests {
         governance_test_utils::advance_epoch(scenario);
         // Advance epoch one more time to distribute some rewards.
         // The delegator should get 7 SUI of rewards.
-        governance_test_utils::advance_epoch_with_reward_amounts(0, 50, scenario);
-
+        governance_test_utils::advance_epoch_with_reward_amounts(0, 49, scenario);
```

Comment:
Off-by-one changes here and below because I slightly refactored reward computation, so rounding changed.